torch/onnx/symbolic_opset9.py

```diff
@@ -1,36 +1,30 @@
+"""This file exports ONNX ops for opset 9.
+
+Opset 9 is supported by ONNX release 1.4.1
+release on 01/23/19
+"""
+
+import functools
 import math
+import sys
 import warnings
-from functools import partial, wraps
-from sys import maxsize as maxsize
 from typing import Optional
 
 import torch
+import torch.nn.modules.utils as modules_utils
```

Comment:
nit: in other files, this is imported without aliasing, and used later by `torch.nn.modules.utils.xxx`. Maybe keep the consistency?